bin/wiki_entity_linking/entity_linker_evaluation.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import random
+
+from tqdm import tqdm
```

Comment:
We just recently ran into a bug with importing `tqdm`, cf https://github.com/explosion/spaCy/pull/4263. The (ugly and temporary) fix is to import it only within the functions themselves, could you update your code? We'll revert it back once the underlying bug is fixed in `tqdm`